docs/dev/devel-env.rst

```diff
@@ -186,6 +186,16 @@ Your development environment is now ready. Now, let's actually see ListenBrainz
 load locally!
 
 
+Install node dependencies
+-------------------------
+
+You also need to install some JavaScript dependencies.
+
+.. code-block:: bash
+
+    docker-compose -f docker/docker-compose.yml -p listenbrainz run --rm web npm install
```

Comment:
If there are build steps after installation (`postinstall` script, like running webpack and such), we'll need to run this command as the LB user.
I assumer we're not risking anything?
`docker-compose -f docker/docker-compose.yml -p listenbrainz run --rm -u listenbrainz web npm install`

Reply:
Out of curiosity, what is the reason for running this command separately and not at some point in the Dockerfile?